integration/spark2/src/main/scala/org/apache/spark/sql/parser/CarbonSparkSqlParser.scala

```diff
@@ -233,6 +233,9 @@ class CarbonHelperSqlAstBuilder(conf: SQLConf,
       case _ =>
         // ignore this case
     }
+    if (partitionFields.nonEmpty && options.isStreaming) {
```

Comment:
maybe it is better to do it in `validateStreamingProperty`, it is invoked in line 213

Reply:
ok